core/App/localization/en/index.ts

```diff
@@ -206,7 +206,7 @@ const translation = {
     "TermsAndConditions": "Terms and conditions",
     "RemoveFromWallet": "Remove from wallet",
     "Revoked": "Revoked",
-    "CredentialRevokedMessageTitle": "Revoked On ",
+    "CredentialRevokedMessageTitle": "This credential is revoked",
```

Comment:
Update in FR and pb-BR?

Reply:
Those strings are already correctly translated. The en string was originally different.